extras/systemd/foreman.socket

```diff
@@ -7,7 +7,8 @@ ListenStream=3000
 # Socket options matching Puma defaults
 NoDelay=true
 ReusePort=true
-Backlog=1024
+# Match systemd v254+'s default because EL9 has 252
```

Comment:
Do our supported Debian's have the same value?

Reply:
I didn't check, but https://packages.debian.org/bookworm/systemd tells me Debian 12 also has systemd 252 (and kernel 6.1 with glibc 2.36). Looks like for this specific case it's in the same boat as EL9 and should also benefit from this.